ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -1212,7 +1212,7 @@ void compact(CompactionDetails details, byte[] bundleReadBuffer) throws IOExcept
     checkStarted();
     compactor.compact(details, bundleReadBuffer);
     checkCapacityAndUpdateReplicaStatusDelegate();
-    logger.trace("One cycle of compaction is completed on the store {}", storeId);
+    logger.info("One cycle of compaction is completed on the store {}", storeId);
```

Comment:
Maybe we can use `DEBUG` level?

Reply:
This is changed to `info` deliberately. When I look the log, there is log recording the start of compaction cycle, but no log for the end of the compaction cycle. 
In addition, this is not a repeated printed log. It should be ok to be in info level.